### ISSUE TITLE
Feature Idea: daytimes to configure on/off, delay, etc.

### ISSUE THREAD

**Issue by jfrux** [benleb/ad-automoli#6]
Needing to enable / disable this functionality during certain times of the day.
Or change the "delay" depending on the time of day...

Just wanted to put this here.  Might be something I attempt in a PR soon...

**Comment by benleb** [comment]
delays per daytimes are not available right now, but should not be hard to implement, I will have a look :)

to disable, just set the `light` option to 0, like here in my office 😄
![image](https://user-images.githubusercontent.com/512997/72347230-f2f17380-36d7-11ea-83a0-aee25b272907.png)

**Comment by jfrux** [comment]
So setting brightness to 0 disables the delay system turn off?
Most of my setups only have switches, not a brightness setting.  I’m guessing your light_setting fix in #5 will fix that and it will all make more sense to me. :D

Thanks!

**Comment by benleb** [comment]
no, 0 disables turning on the lights in that room for that time. I will implement a delay depending on the daytime this week :)

**Comment by jfrux** [comment]
No biggie!
I may feel inclined to PR shortly… trying to get a stable system in general at this time… switching away from Hass.io and just managing it all inside of docker-compose is blazing fast in comparison from my findings.

**Comment by tjedrak** [comment]
@benleb and can we add also option `delay: 0`, so for that the light won't turn off again in specific daytime?

so let's say we have for example this config:

```
 delay: 90
 daytimes:
    - { starttime: "05:30", name: morning, light: "Morning" }
    - { starttime: "07:30", name: day, light: 0 }
    - { starttime: "20:30", name: evening, light: 90, delay: 0 }  // this overwrites top-level delay and never turn off light untill 22:30
    - { starttime: "22:30", name: night, light: 20 }
    - { starttime: "23:30", name: more_night, light: 0 }
```

**Comment by benleb** [comment]
sounds good! 
so todos for me/this issue are: 
* implement per daytime delay
* disable delay with value '0'
right?

sorry, was sick many days now and not had the desire to code :/

**Comment by tjedrak** [comment]
Yes correct!
- global delay
- delay per daytime (overrides global)
- delay off = 0 per daytime

No worries hope you feel better now :)

> sounds good!
> so todos for me/this issue are:
> 
> * implement per daytime delay
> * disable delay with value '0'
>   right?
> 
> sorry, was sick many days now and not had the desire to code :/

**Comment by benleb** [comment]
disable delay with value '0' implemented in 40c8ad8
working on per daytime delays now :)

**Comment by benleb** [comment]
per-daytime delays implemented in c28ae24

both changes now available via [v0.5.3](https://github.com/benleb/ad-automoli/releases/tag/v0.5.3)
